sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/schema.yaml

```diff
@@ -0,0 +1,215 @@
+  mode: NULLABLE
+- description: The value of the `mozilla.partner.id` preference in the Partner Distribution Configuration File.
+  name: partner_id
+  type: STRING
+  mode: NULLABLE
+- description: The anonymized path of the engine xml file. For details on the components refer to the metadata for telemetry.new_profile.
+  name: engine_data_load_path
+  type: STRING
+  mode: NULLABLE
+- description: The name of the default search engine.
+  name: engine_data_name
+  type: STRING
+  mode: NULLABLE
+- description: 'The origin of the search engine. The value will be default for engines that are built-in or from distribution partners,
+  verified for user-installed engines with valid verification hashes, unverified for non-default engines without verification hash,
+  and invalid for engines with broken verification hashes.'
+  name: engine_data_origin
+  type: STRING
+  mode: NULLABLE
+- description: The HTTP url we would use to search. For privacy, we don’t record this for user-installed engines.
+  name: engine_data_submission_url
+  type: STRING
+  mode: NULLABLE
+- description: The model IDs for Apple desktop devices. Applies to Mac only.
+  name: apple_model_id
+  type: STRING
+  mode: NULLABLE
+- description: City retrieved as a result of a geographic lookup based on the client's IP address.
+  name: city
+  type: STRING
+  mode: NULLABLE
+- description: First major country subdivision, typically a state, province, or county.
+  name: subdivision1
+  type: STRING
+  mode: NULLABLE
+- description: The Firefox channel, set to Other for unrecognized channel names.
+  name: normalized_channel
+  type: STRING
+  mode: NULLABLE
+- description: The ISO 3166-1 alpha-2 country code.
+  name: country
+  type: STRING
+  mode: NULLABLE
+- description: The OS name, set to Other for unrecognized OS names.
+  name: normalized_os
+  type: STRING
+  mode: NULLABLE
+- description: The OS version.
+  name: normalized_os_version
+  type: STRING
+  mode: NULLABLE
+- description: 'How the profile was selected during startup. Possible reasons are:
+   unknown: Generally should not happen, set as a default in case no other reason occurred. profile-manager: The profile was selected by the profile manager.
+   profile-reset: The profile was selected for reset, normally this would mean a restart. restart: The user restarted the application,
+   the same profile as previous will be used. argument-profile: The profile was selected by the --profile command line argument.
+   argument-p: The profile was selected by the -p command line argument. firstrun-claimed-default: A first run of a dedicated profiles build chose the old
+   default profile to be the default for this install. firstrun-skipped-default: A first run of a dedicated profiles build skipped over the old default profile
+   and created a new profile. restart-claimed-default: A first run of a dedicated profiles build after a restart chose the old default.'
+  name: startup_profile_selection_reason
```

Comment:
cc @SuYoungHong @irrationalagent @Marlene-M-Hirose @gkabbz revisit this, seems like this is "missing" from main ping clients

Reply:
Good call out. Just to clarify, the data does exist in main pings (`telemetry.main`), however, it's not propagated to `telemetry.clients_daily`, which is where this ETL is pulling from.